about.html

```diff
@@ -0,0 +1,129 @@
+           <a href="index.html">Home</a>
+            <a href="#contact">Contact</a>
+            <a href="#live">Live</a>
+      </na>
+    </div>
+    <header class="about-header">
+      <a class="hum-burger"><i class="fa-solid fa-bars"></i></a>
+    </header>
+
+ <section class="desktop-nav">
+
+          <div class="social-links">
+            <a href="#"><i class="fa-brands fa-facebook"></i></a></a>
+          <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+          <a href="#">Logout</a>
+          <a href="#">myPage</a>
+          <a href="#">Korean</a>
+        </div>
+        <div class="logo-container">
+          <a href="index.html"><img src="images/bg-visit.jpg" alt="logo" class="logo"></a>
+          
+          <ul class="info-lists">
+              <li><a href="about.html" class="red-about">About</a></li>
+             <li><a href="">Program</a></li>
+             <li><a href="">Join</a></li>
+              <li><a href="">Sponsor</a></li>
+               <li><a href="">News</a></li>
+                <li class="red-border"><a href="">CC Campaign</a></li>
+            </ul>
+        </div>
+ </section>
+
+
+    <section class="home-page">
+      <div class="description">
+        <h4 class="heading2">"Hello, Tourist!"</h4>
+        <h1 class="home-heading">
+          Visit Oromia, Ethiopia Land of Origin and Heritages in 2023!
+        </h1>
+        <p class="info info-about">
+          Oromia is a largest region in Ethiopia with potential natural
+          resources and cultural heritages. Visit Oromia where the sheer
+          diversity of the culture, people and landscape is sure to leave you
+          astounded.
+        </p>
+
+        <div class="contact-us" id="contact">
+          <p class="info">
+            Please contact-us for further explanation about visit oromia @
+            <br /><span class="creative-heading">VisitOromia@2023</span>
+          </p>
+        </div>
+      </div>
+    </section>
+    <section class="creative-container">
+        <h2 class="creative-heading">
+          Creative Commons Visit Oromia Logo in 2023!
+        </h2>
+        <hr />
+        <p class="about-info info">
+          The logo of visit Oromia was developed by the group of software
+          developer  who were qualified by visit-oromia tourism commission. <br> After
+          careful consideration of the process of selection though online boot
+          voting  we are glad to announce that Roba as a winner for the oromia
+          tourism logo winner.
+        </p>
+    </section>
+    <div class="logo-about logo-container">
+      <img src="images/bg-visit.jpg" alt="" />
+    </div>
+    <section class="live-oromia" id="live">
+      <div class="see-people">
+        <h2 class="creative-heading">
+          See Previous Visit Oromia Gallery in 2010 and 2016
+        </h2>
+        <p>Look at the people from Land of Oromia, Ethiopia in the pasts</p>
+      </div>
+
+      <div class="live-cont">
```

Comment:
- [ ] As requested by previous reviewers, kindly add spaces to the left and right of this container in the desktop view. Consider adding a padding left and right of 200px to the destop view to match the design template.
### Your design
![image](https://user-images.githubusercontent.com/94551484/217919448-5a833708-4111-4552-8c0c-2ec12962e50b.png)

### Design template
![f2](https://user-images.githubusercontent.com/94551484/217919576-a21852ad-3267-447f-bb6c-c3e62d6c018d.png)